### ISSUE TITLE
Enter one-line summaryloading input method appears in recent apps list when eyes-free keyboard is enabled

### ISSUE THREAD

**Issue by GoogleCodeExporter** [clc/eyes-free#77]
```
What steps will reproduce the problem?
1. Set the eyes-free keyboard as the default input method.
2. Reboot the phone.
3. Long press home to bring the list of running aplications.
You will have an app called "loading input method" in the list of running apps.
What is the expected output? What do you see instead? I expect to have an empty 
list when my phone first boots. Instead, I have an app that will just return 
you to the home screen if you select it.


What version of the product are you using? On what operating system?  Eyes-free 
Keyboard version 1.01 on a device running Android 2.3.


Please provide any additional information below.
This issue will also occurr if I switch the default input method to talkback 
keyboard.

```

Original issue reported on code.google.com by `hittsj...@gmail.com` on 3 Dec 2011 at 7:08

**Comment by GoogleCodeExporter** [comment]
```
BootActivity is missing android:excludeFromRecents attribute.
```

Original comment by `alanv@google.com` on 6 Dec 2011 at 11:47
- Added labels: **Priority-Low**
- Removed labels: **Priority-Medium**

**Comment by GoogleCodeExporter** [comment]
```
Resolved in v1.1.1 update.
```

Original comment by `alanv@google.com` on 17 Dec 2011 at 6:01
- Changed state: **Fixed**
